gef-ui/templates/includes/application-preview/task-comments.njk

```diff
@@ -1,43 +1,51 @@
 {% from "moj/components/banner/macro.njk" import mojBanner %}
-
-{% if isUkefReviewAvailable or comments.length > 0 %}
+{% if isUkefReviewAvailable or unissuedFacilitiesPresent or facilitiesChangedToIssued.length > 0 or comments.length > 0 %}
```

Comment:
Server UI function as condition is slightly complicated for a NJK template.

Reply:
Yes can we make this a variable in the server and pass simple one word variable into the template?

Reply:
simplified